### ISSUE TITLE
Tray icon not displaying image

### ISSUE THREAD

**Issue by akmalFir** [IsmaelMartinez/teams-for-linux#506]
**Describe the bug**
Tray icon does not showing any image. There is icon but just no image

**To Reproduce**
On opening the application it will happen

**Screenshots**
![Screenshot from 2021-12-11 22-14-28](https://user-images.githubusercontent.com/82021721/145680048-8c7ce9df-25cf-4416-ab5b-edef2ba79ce5.png)
The second icon from left is teams icon

**Desktop (please complete the following information):**
 - OS: Zorin os 16
 - Installation package: flatpak
 - Version: 1.0.20

**Comment by sean-gray** [comment]
Same issue in KDE neon 

OS: KDE neon User - Plasma 25th Anniversary Edition x86_64 
Kernel: 5.11.0-43-generic 
Packages: 1986 (dpkg), 15 (flatpak), 8 (snap) 
Shell: bash 5.0.17 
Resolution: 2560x1440, 2560x1440 
DE: Plasma 
WM: KWin 
Theme: Breeze Light [Plasma], Breeze [GTK2/3] 
Icons: [Plasma], breeze-dark [GTK2/3] 
Terminal: konsole 
![Screenshot_20211215_141400](https://user-images.githubusercontent.com/35218853/146250723-599af9fc-c64d-4ddd-bcf8-ceb85aa75e19.png)

**Comment by IsmaelMartinez** [comment]
Hi,

I suspect then is some sort of electron-builder config that ain't working.
Can you check if the tray icon was there in an older version? Like 1.0.7
(so using the old electron version)

Thanks and apologies, I am not even in Linux so i can try to help with
advice or ideas but no much else.

**Comment by sean-gray** [comment]
@IsmaelMartinez Its actually working for the snap and appimage versions of the app correctly. Its only in flatpak version that its not showing. My guess is flatpak requires something extra. I will keep researching this.

**Comment by jijojosephk** [comment]
@sean-gray @IsmaelMartinez Looks like the icon should be named as the flatpak app Id. Not sure but can give a try.

**Comment by jijojosephk** [comment]
Renaming the icon didn't work

**Comment by jijojosephk** [comment]
@akmalFir @sean-gray Are you guys using custom themes, icons?

**Comment by sean-gray** [comment]
@jijojosephk no. Just KDE Breeze theme and Icon that is default in for KDE. I have tried other DE and its the same.

**Comment by 95A31** [comment]
Same problem here. I am on a fresh installed Manjaro with KDE.
![image](https://user-images.githubusercontent.com/3296448/148299034-fbca7e44-4ebe-4492-87ac-5deefaa96aad.png)

**Comment by darrellenns** [comment]
Not sure if it's related, but is there supposed to be an unread messages indication on the tray icon?

**Comment by micwoj92** [comment]
@darrellenns yes
![image](https://user-images.githubusercontent.com/45581170/151610044-6d7c855d-3f9a-4c1f-9789-7a3b45e3a5a2.png)

**Comment by darrellenns** [comment]
Interesting - I am on Arch and had "teams" installed from AUR as well as teams-for-linux installed from flathub. I was not getting the indicator in the tray because teams-for-linux was actually using the tray icon from "teams". I removed the AUR-installed "teams", and now I have the missing tray icon issue. So yes, it seems that the missing tray icon issue is what I was experiencing as well.

**Comment by darrellenns** [comment]
Also I am on gnome, so it's not just kde that is affected.

**Comment by FinixFighter** [comment]
I have the same issue in a Linux Mint 20.3 (Xfce version)

**Comment by IsmaelMartinez** [comment]
I think the unread messages is a MS bug they introduced in the browser version. read in https://github.com/IsmaelMartinez/teams-for-linux/issues/536 for more info.

**Comment by jijojosephk** [comment]
This has something to do with icon theming. #536
